frontend/server/src/Controllers/School.php

```diff
@@ -226,7 +226,43 @@ public static function apiUsers(\OmegaUp\Request $r): array {
     }
 
     /**
-     * Gets the top five schools that are showed on the index page.
+     * Gets the top X schools of the month
+     * @return list<array{school_id: int, name: string, country_id: string, score: float}>
+     */
+    private static function getTopSchoolsOfTheMonth(
+        int $rowcount,
+        bool $canUseCache
```

Comment:
nit:

```suggestion
        bool $useCache
```

aquí le estás indicando a este método que use el cache.